### PR TITLE
Feature/dash wallet

### PR DESCRIPTION
**Pull request by synchuk2010**
#### Description
- Add Dash wallet
- Update readme

### CODE REVIEW

**Comment by kozlov-oleksandr** [comment]
@synchuk2010 , PR must have description. Pls, write it.
Commit with unit tests must include label `[UT]` at the beginning of the commit message.
Update PR, pls

File: essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        static var getTransactionById: NSString = "/dash/transactions/%@"
+        static var sendTransaction: String = "/dash/wallets/transactions"
+    }
+    
+    enum Headers {
+        static var address = "address"
+        static var data = "data"
+    }
+    
+    enum Body {
```
Comment (impul) [inline essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift]: Unused enum

File: essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    enum Path {
+        static var getBalance: NSString = "/dash/wallets/%@/balance"
+        static var getUTxo: NSString = "/dash/wallets/%@/utxo/"
+        static var getTransactionsHistory: NSString = "/dash/wallets/%@/transactions"
+        static var getTransactionById: NSString = "/dash/transactions/%@"
+        static var sendTransaction: String = "/dash/wallets/transactions"
+    }
+    
+    enum Headers {
+        static var address = "address"
```
Comment (impul) [inline essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift]: Unused ` static var address = "address"`

**Review by impul — CHANGES_REQUESTED** [review]

File: essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift (hunk 1 above)
Comment (synchuk2010) [reply essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift]: @impul, removed 99314a1

**Comment by impul** [comment]
@synchuk2010, please mark conversations as Resolved after you do a changes

File: essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift (hunk 2 above)
Comment (synchuk2010) [reply essentia-bridges-api-ios/Wallets/Dash/DashEndpoint.swift]: @impul, removed 99314a1

**Review by impul — APPROVED** [review]

**Pull request merged**